external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/NamedSubscription.java

```diff
@@ -56,6 +57,6 @@ public NamedSubscription(String ... topics) {
 
     @Override
     public String getTopicsString() {
-        return String.valueOf(topics);
+        return StringUtils.join(topics, ",");
```

Comment:
I would like to suggest that you use instead Guava's `Joiner.on(",").join(topics);` which will avoid adding a new dependency